src/DotNet.Consolidate/Program.cs

```
@@ -21,17 +22,36 @@ private static void HandleParseError(IEnumerable<Error> errors)
             }
         }
 
+        private static ILogger GetLogger(Options options)
+        {
+            switch (options.OutputFormat)
+            {
+                case OutputFormat.Json:
+                    return new JsonLogger();
+                case OutputFormat.Text:
+                    return new Logger();
+                default:
+                    return new Logger();
```

We should throw an exception here.

---

Sure.